vector/src/main/java/im/vector/app/features/home/room/list/RoomSummaryPagedController.kt

```diff
@@ -57,8 +65,14 @@ class RoomSummaryPagedController(
     }
 
     override fun buildItemModel(currentPosition: Int, item: RoomSummary?): EpoxyModel<*> {
-        // for place holder if enabled
-        item ?: return RoomSummaryItemPlaceHolder_().apply { id(currentPosition) }
-        return roomSummaryItemFactory.create(item, roomChangeMembershipStates.orEmpty(), emptySet(), displayMode, listener)
+        return if (item == null) {
+            val host = this
+            RoomSummaryItemPlaceHolder_().apply {
```

Comment:
Nit: `roomSummaryItemPlaceHolder{ }` could be used
```suggestion
            roomSummaryItemPlaceHolder {
```

Reply:
`roomSummaryItemPlaceHolder` adds model to collector, but here we need to return item instead, so we can't really use this inside `buildItemModel` method :(

Reply:
Oh, true, my bad.